scripts/benchmark.cr

```diff
@@ -2,23 +2,31 @@
 
 require "../src/pluto"
 
-record Result, name : String, time : Int64, memory : Int64
+record Result, name : String, time : Float64, memory : Int64
 
-def benchmark_memory
+def benchmark_memory(&)
   bytes_before_measure = GC.stats.total_bytes
   yield
   (GC.stats.total_bytes - bytes_before_measure).to_i64
 end
 
-def benchmark_time
+def benchmark_time(&)
   time = Time.monotonic
   yield
   Time.monotonic - time
 end
 
+private def elapsed_text(millis)
+  return "#{millis.round(2)}ms" if millis >= 1
+
+  return "#{(millis * 1000).round(2)}µs" if millis >= 0.001
+
+  "#{(millis * 1_000_000).round(2)}ns"
+end
```

Comment:
Added this to nicely format smaller time increments. Crop operation runs on the micro seconds scale :D